gateway/rpc_storage_handler.go

```diff
@@ -1144,6 +1149,17 @@ func (r *RPCStorageHandler) ProcessKeySpaceChanges(keys []string, orgId string)
 	r.Gw.MainNotifier.Notify(n)
 }
 
+// Function to handle fallback deletion using token ID
+func (r *RPCStorageHandler) deleteUsingTokenID(key, orgId string, resetQuota bool, status int) (int, error) {
+	if status == http.StatusNotFound {
+		id, err := storage.TokenID(key)
+		if err == nil {
+			_, status = r.Gw.handleDeleteKey(id, orgId, "-1", resetQuota)
+		}
+	}
+	return status, nil
+}
+
```

Comment:
with this we will handle situations where a key is requested to be deleted using the base 64 representation, but it's stored using the custom id, Eg: `apikey-customID`